### PR TITLE
Multiple booth add

### PR DESCRIPTION
**Pull request by Husso1997**

### CODE REVIEW

File: src/app/admin/booth/add-booth/add-booth.component.html [hunk 1]
```diff
@@ -16,8 +16,12 @@
...
       <input class="form-control" type="number" placeholder="Mængde af stande" formControlName="amount">
+      <small class="form-text text-muted">
+        Et maximum af {{maxAmountOfBoothsAllowed}} stande kan blive lavet ad gangen.
+      </small>
     </div>
 
     <button class="btn btn-primary" type="submit">Tilføj stande</button>
   </form>
+
 </div>
```
Comment (asbamse) [inline src/app/admin/booth/add-booth/add-booth.component.html:29]: Missing:
<div *ngFor="let alert of alerts">
    <alert [type]="alert.type" [dismissOnTimeout]="alert.timeout" (onClosed)="onAlertClosed(alert)" [dismissible]="true"><strong>{{ alert.msgStrong }}</strong> {{ alert.msg }}</alert>
  </div>

File: src/app/admin/booth/add-booth/add-booth.component.ts [hunk 2]
```diff
@@ -26,21 +26,20 @@ export class AddBoothComponent implements OnInit {
   addBooth() {
     let boothForm:any = this.boothForm.value;
 
-    for (let i = 0; i < boothForm.amount; i++)
-    {
+    if(boothForm.amount > this.maxAmountOfBoothsAllowed) {  }
```
Comment (asbamse) [inline src/app/admin/booth/add-booth/add-booth.component.ts]: Missing:
error => {
              this.alerts.push({
                class: "text-center",
                type: "danger",
                msgStrong: "Fejl!",
                msg: "Et maximum af " + maxAmountOfBoothsAllowed + " stande kan blive lavet ad gangen.",
                timeout: 5000
              });

**Review by asbamse — CHANGES_REQUESTED** [review]
You should add errors to view and refresh the list on addition. It would be nice with some confirmation messaging.

**Review by asbamse — APPROVED** [review]

**Comment by Applemelon** [comment]
very nice

**Pull request merged**